token_server/README.md

```diff
@@ -1,4 +1,32 @@
 # Simple web server to provide EarthEngine tokens to non-whitelisted users
 
-Run with `node --experimental_modules token_server/token_server.js`.
+## Deploy to Google App Engine
 
+* Follow initial steps for Google App Engine deployment: install `gcloud`, log
+in as the GD user `gd-earthengine-user@givedirectly.org`, and set the gcloud
+project to `mapping-crisis`. Helpful resource:
+[NodeJS Quickstart](https://cloud.google.com/appengine/docs/standard/nodejs/quickstart).
```

Comment:
Is there anything specific on this page that's relevant to the set up process you're documenting here? I think most/all of it has already been done and isn't pertinent? I agree it's helpful to link to some docs but would be awesome to link to some more specifically relevant places.

Reply:
I couldn't find anything more specifically relevant. But everything I did here came from that as a starting point. However, you're right, there's not much there. Removed.